man/systemd-system.conf.xml

```diff
@@ -127,7 +127,8 @@
 
       <varlistentry>
         <term><varname>RuntimeWatchdogSec=</varname></term>
-        <term><varname>ShutdownWatchdogSec=</varname></term>
+        <term><varname>RebootWatchdogSec=</varname></term>
+        <term><varname>KexecWatchdogSec=</varname></term>
```

Comment:
we have a bus function called "KExec", and we probably should use the same spelling here, i.e. "KExecWatchdogSec"

Reply:
makes sense, done